oat-sa/extension-tao-outcomeui
# Feature/re 69/add trace variables to csv export

**Related to task**: https://oat-sa.atlassian.net/browse/RE-69
**Description**: Trace variables export in CSV export

**How to test**:
1. create delivery and pass the test
2. to generate trace variables you can ctrl+tab to other tab in browser or enter/exit full screen in browser
3. open Export Table in Result tab
4. click 'Add all trace variables'
5. click 'Export CSV file'

**Expected Result**: CSV file should be loaded. This file should contain 'Trace variables' column with trace variables in json format.
